src/main/java/de/cerus/jdasc/interaction/followup/FollowupMessage.java

```diff
@@ -11,13 +11,15 @@
     private final String content;
     private final boolean tts;
     private final List<MessageEmbed> embeds;
+    private final int flags;
 
     public FollowupMessage(final String content,
                            final boolean tts,
-                           final List<MessageEmbed> embeds) {
+                           final List<MessageEmbed> embeds, int flags) {
```

Comment:
Please put the 'flags' parameter into the next line (like all the other parameters). I'm sorry, but I'm a huge fan of consistency :D

Reply:
Fixed :D